src/views/ErrorPage.tsx

```diff
@@ -0,0 +1,20 @@
+import { useRouteError } from 'react-router-dom';
+
+type RouteError = { statusText: string; message: string };
+
+const ErrorPage = () => {
+  const error = useRouteError() as RouteError;
+  console.error(error);
```

Comment:
consider prefacing the output to indicate that it's coming from this component

Reply:
Done